Sources/PaymentButtons/PayPalButton.swift

```diff
@@ -113,69 +113,65 @@ public final class PayPalButton: PaymentButton {
     deinit {}
 }
 
-public extension PayPalButton {
```

Comment:
I am not opposed to this change if we decide on a better name, but it seems like using `Representable` is a common practice for SDKs that offer both SwiftUI and UIKit. For example, here is Stripe's example SwiftUI view for their card form: https://github.com/stripe/stripe-ios/blob/392f6d08fcb4f075c5f36a376c14477e7087a6e3/Example/UI%20Examples/UI%20Examples/Source/SwiftUICardFormView.swift#L19.

Not saying that we _have_ to go with this option but this is why we originally went with using `Representable`.

Reply:
I personally have not seen `.Representable` used before. I usually expect a view to be self-contained rather than being defined within a class. I'm curious if there are more examples of this pattern. I have not seen any aside from our's and the Stripe example linked above.

Reply:
There aren't a ton of SDKs that offer both a SwiftUI and UIKit UI element out of the box. When we originally went with using `Representable` we attempted to find other examples but Stripe was the only one we were able to find (open source and not just a question on StackOverflow) that offered 1 elements for both UIKit and SwiftUI.

Apple Pay _just_ added a button for SwiftUI with iOS 16+ (`PayWithApplePayButton` - SwiftUI and `PKPaymentRequest` - UIKit). This could be the pattern that we move forward with but at the time we went with `Representable` only 1 SDK (Stripe) offered UIKit + SwiftUI elements at all, so we went forward with the approach that existed at the time.

We also originally attempted to subclass the main button class with `UIViewRepresentable`, but there was apparently a bug that allowed us to do this originally (PR #84) for context to fix this and move to `Representable`. That to say, I am not opposed to whatever approach we decide but at the time of writing this code originally we had very few examples to base our code on. 😄 

Do you have any examples of an SDK using 1 namespace for both their UIKit and SwiftUI elements? It certainly _could_ be possible again but basing my knowledge off of the work done over a year and half ago when SwiftUI was still quite new.

Reply:
`Representable` has worked well for us up to this point, but I do think it makes sense to revisit this until it feels 100% right.

SwiftUI and other declarative UI frameworks put a premium on readability. I'd vote for reserving the simplest name for SwiftUI like Apple does with `PayWithApplePayButton` <-> `PayPalButton`.

I also like the PK (Payment Kit) module prefix they use for the UIKit side of things. Maybe we could do something similar like `PPPayPalButton` or `PYPayPalButton`?

Reply:
@jaxdesmarais Absolutely that makes sense and there is nothing wrong with the `.Representable` approach. However, I feel if we can come up with a better name to delineate the two, I think that can be considered best practice in today's SwiftUI environment.

I like the idea @sshropshire suggested. For the UIView, we can perhaps prefix the class names (with a non-Apple prefix) and leave the SwiftUI view with no prefix. For example:

UIKit: `PPPayPalButton` (or something similar)
SwiftUI: `PayPalButton`

I'd like to hear @scannillo's thoughts on the prefix idea because she did mention this:

https://github.com/paypal/paypal-ios/pull/84#issuecomment-1291422816

Reply:
Yep, like I said above, I am by no means saying using `Representable` is the right approach now. Just that it _was_ the right approach at the time this code was written. Since there was some debate on the initial "why" I was just attempting to provide context from that point in time since most of the current team wasn't around when that decision was made. :)

Reply:
I'm open to revisiting this for when we have an idea on a name. For now, the way it works is perfectly fine.

Reply:
Same, I'll revert it back to `.representable` and start a conversation on Slack since what we decide will impact other parts of the codebase where we build the UI in UIKit and wrap it to display in SwiftUI view